sdk/containerregistry/Azure.Containers.ContainerRegistry/api/Azure.Containers.ContainerRegistry.netstandard2.0.cs

```diff
@@ -144,6 +143,14 @@ public enum ManifestOrderBy
         LastUpdatedOnDescending = 1,
         LastUpdatedOnAscending = 2,
     }
+    public partial class ManifestWriteableProperties
+    {
+        public ManifestWriteableProperties() { }
+        public bool? CanDelete { get { throw null; } set { } }
+        public bool? CanList { get { throw null; } set { } }
+        public bool? CanRead { get { throw null; } set { } }
+        public bool? CanWrite { get { throw null; } set { } }
```

Comment:
Would this always be true?

Reply:
The service will return the value it has for these - true or false.  These are nullable so callers don't inadvertently write over existing values by taking a default here.

Reply:
The reason I ask is that as a property of a type called `xWriteableProperty` this property seems counterintuitive. I'd expect `CanWrite` to be true by definition.

Reply:
Ah, I see.  The writeable properties are properties of the x -- i.e. that you can write to an artifact or a repository.  Do you think a different name would express this more clearly?

Reply:
I think I'd need to understand more about the e2e scenario for how `ManifestWriteableProperties` are consumed and modified as it relates to these Can* properties in order to come up with a suggestion :).

Reply:
The classic example is that a service is doing a deployment of an image, which takes hours or days, and they want to prevent someone else from updating the image during that time.

https://gist.github.com/annelo-msft/e0d4976b029e8815dcbcf5faf449db32#2-make-tagged-image-read-only